src/header/keys.rs

```diff
@@ -45,7 +44,7 @@ impl RoutingKeys {
     // or should this be renamed to 'new'?
     // Given that everything here except RoutingKeys lives in the `crypto` module, I think
```

Comment:
Note to line 40: `pub blinding_factor: BlindingFactor,` => I wonder if it would make sense to just change its type to a private key/scalar and not bother with constant bytes conversion

Reply:
It's used as a `StaticSecret` so I can change that yes

Reply:
Refactored in https://github.com/nymtech/sphinx/pull/94/commits/6899145f26fbab13afaa84e5d303e29ede919c80